pkgs/tools/security/sshuttle/default.nix

```diff
@@ -10,17 +10,19 @@
 
 python3Packages.buildPythonApplication rec {
   pname = "sshuttle";
-  version = "1.0.3";
+  version = "1.0.5";
 
   src = python3Packages.fetchPypi {
     inherit pname version;
-    sha256 = "0fff1c88669a20bb6a4e7331960673a3a02a2e04ff163e4c9299496646edcf61";
+    sha256 = "fd8c691aac2cb80933aae7f94d9d9e271a820efc5c48e73408f1a90da426a1bd";
   };
 
   patches = [ ./sudo.patch ];
 
   nativeBuildInputs = [ makeWrapper python3Packages.setuptools-scm ];
 
+  propagatedBuildInputs = [ python3Packages.psutil ];
```

Comment:
We can fetch https://github.com/sshuttle/sshuttle/commit/c06581cf83fdbf34269a3a3b42daa28b55ae4129 to avoid the extra dependency.

Reply:
Done.

Reply:
I backed out of that change because it broke `nix-build -A telepresence`.